### ISSUE TITLE
readsafely not defined

### ISSUE THREAD

**Issue by axsk** [JuliaIO/JLD.jl#84]
``` julia
using JLD
JLD.readsafely
```
returns a `ERROR: UndefVarError: readsafely not defined`.
This happens for me on `Julia 0.4.6` with the latest `JLD` release as well as `master`.

**Comment by timholy** [comment]
Have you run into a situation where you think this might help you? In other words, I'm wondering if I should remove it from the docs or re-implement it. It's currently in JLD00.jl but not JLD.jl.

**Comment by axsk** [comment]
Not realy. I just tried it because I am having problems reading `.jld` files I saved 2 weeks ago:
``` julia
ulia> JLD.load("jul16/pl1s0.1r0.00033981209046252456t1atrue.jld")
WARNING: UndefVarError(:load)
 in load at /home/numerik/bzfsikor/.julia/v0.4/FileIO/src/loadsave.jl:82
 in load at /home/numerik/bzfsikor/.julia/v0.4/FileIO/src/loadsave.jl:45
ERROR: UndefVarError: load not defined
 in load at /home/numerik/bzfsikor/.julia/v0.4/FileIO/src/loadsave.jl:82
 in load at /home/numerik/bzfsikor/.julia/v0.4/FileIO/src/loadsave.jl:45
```

**Comment by timholy** [comment]
That's really weird. You might want to insert a `@show Library` at that spot in FileIO and see what's going on.

**Comment by axsk** [comment]
Since back then I multiple times ran into situations where I needed to just quickly read in some data from an old version of a type but there was no simple way. Of course the readas trick works, but it is still quite cumbersome if used in a runtime environment, where I merely need to read in old data a single time, e.g. for manual conversion or comparison.
I see that defining the proper translate/readas is the proper way for handling these problems inbetween published package versions, but I think it is overkill for inbetween work.

TL;DR: I opt for reimplementation.
